vdt/versionplugin/buildout/shared.py

```diff
@@ -96,7 +96,9 @@ def requirement_versions(self):
             # when comes_from is not set it is a dependency to ourselves. So
             # skip that
             if install_req.comes_from:
-                versions[install_req.name] = install_req.req.specs[0][1]
+                versions[install_req.name] = ""
+                if install_req.req.specs:
+                    versions[install_req.name] = install_req.req.specs[0][1]
```

Comment:
why not try except? You dont know how pip behaves so could be other errors. Do want to know about it or do you want the tool to do something useful in case of weirdness?